libs/clients/zebpay/client.go

```diff
@@ -248,25 +253,33 @@ func affixAccessToken(req *http.Request, opts *ClientOpts) error {
 
 // CheckTransfer uploads the bulk payout for gemini
 func (c *HTTPClient) CheckTransfer(ctx context.Context, opts *ClientOpts, id uuid.UUID) (*CheckTransferResponse, error) {
+	var resp = new(CheckTransferResponse)
+	resp.ID = id
```

Comment:
Not nice. This is better:

```suggestion
	resp := &CheckTransferResponse{ID: id}
```